pages/index.jsx

```diff
@@ -3,25 +3,30 @@ import styled from "styled-components";
 import AddIcon from "@/components/AddIcon";
 
 export default function HomePage({ games }) {
+  const progressSections = ["In Progress", "Planned", "Completed"];
+
+  function isSectionEmpty(progress) {
+    return games.some((game) => game.progress === progress);
+  }
+
   return (
     <>
       <StyledMain>
         <h1>Game Flow</h1>
         <StyledPageHeadline>Game List</StyledPageHeadline>
-        <section>
-          <StyledListLabel>In progress</StyledListLabel>
-          <GameList progressLabel={"In Progress"} games={games} />
-        </section>
-        <section>
-          <StyledListLabel>Planned</StyledListLabel>
-          <GameList progressLabel={"Planned"} games={games} />
-        </section>
-        <section>
-          <StyledListLabel>Completed</StyledListLabel>
-          <GameList progressLabel={"Completed"} games={games} />
-        </section>
+        {/* Renders a list with a label for each progress status */}
+        {progressSections.map((progress) => {
+          return (
+            <section key={progress}>
+              <StyledListLabel>{progress}</StyledListLabel>
+              <GameList progressLabel={progress} games={games} />
+              {!isSectionEmpty(progress) && (
+                <p>No games at the moment, please add one!</p>
+              )}
```

Comment:
Just a suggestion (you don't have to do it):
To make the code a bit cleaner here you can move this logic to the GameList component. The GameList component filters the games array and could either render the list or render this message (in case of `filteredGames.length===0`.